middlewares/validators/user.js

```diff
@@ -22,7 +22,7 @@ const updateUser = async (req, res, next) => {
         .optional()
         .min(4)
         .max(20)
-        .regex(/^[a-zA-Z0-9]+$/)
+        .regex(/^[a-zA-Z0-9-]+$/)
```

Comment:
Can we define this regex in a variable and use it, it would be descriptive

Reply:
I think no need to add in variable as it is not use anywhere and it is self explanatory